bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/connection/OpenWeatherMapConnection.java

```diff
@@ -38,8 +39,10 @@
 import org.openhab.binding.openweathermap.internal.dto.OpenWeatherMapJsonHourlyForecastData;
 import org.openhab.binding.openweathermap.internal.dto.OpenWeatherMapJsonUVIndexData;
 import org.openhab.binding.openweathermap.internal.dto.OpenWeatherMapJsonWeatherData;
+import org.openhab.binding.openweathermap.internal.dto.onecall.OpenWeatherMapOneCallAPIData;
+import org.openhab.binding.openweathermap.internal.dto.onecallhist.OpenWeatherMapOneCallHistAPIData;
 import org.openhab.binding.openweathermap.internal.handler.OpenWeatherMapAPIHandler;
-import org.openhab.core.cache.ByteArrayFileCache;
+import org.openhab.binding.openweathermap.internal.utils.ByteArrayFileCache;
```

Comment:
The `ByteArrayFileCache``is now available in OHC. Please revert this change.

Reply:
Done. I got ClassNotFoundExceptions for my last update, this is why I added it originally. Looks like my 3.0 snapshot was too old then.